components/batch-system/tests/cases/batch.rs

```diff
@@ -26,6 +27,8 @@ fn test_batch() {
         })))
         .unwrap();
     assert_eq!(rx.recv_timeout(Duration::from_secs(3)), Ok(1));
+    // sleep to wait Batch-System to finish calling end().
```

Comment:
Why need to wait?

Reply:
This is a mistake.

Reply:
Because if we do not wait, this test may fail because it deals with two message in once `poll`.